Makefile

```diff
@@ -42,15 +47,15 @@ install-openvino-test:
 	pip install -e .[openvino]
 	pip install -r tests/openvino/requirements.txt
 	pip install -r tests/cross_fw/install/requirements.txt
-	pip install -r examples/experimental/openvino/bert/requirements.txt
-	pip install -r examples/experimental/openvino/yolo_v5/requirements.txt
 	pip install git+https://github.com/openvinotoolkit/open_model_zoo.git@dcbf53280a95dae3c6538689bafe760470f08ec2#subdirectory=tools/model_tools
 
 install-openvino-dev: install-openvino-test install-pre-commit install-pylint
-	pip install -r examples/post_training_quantization/openvino/mobilenet_v2/requirements.txt
-	pip install -r examples/post_training_quantization/openvino/quantize_with_accuracy_control/requirements.txt
-	pip install -r examples/post_training_quantization/openvino/yolov8/requirements.txt
-	pip install -r examples/post_training_quantization/openvino/yolov8_quantize_with_accuracy_control/requirements.txt
+	pip install -r examples/experimental/openvino/bert/requirements.txt
```

Comment:
Thanks for the comment! I absolutely agree that this is potential point of getting a broken environment. In this PR I removed example's requiremnets from install-`backend`-test to avoid that case in a test environment. In future we should think about how to rework makefile to avoid such case in the dev environment as well but support pylint check in the same time.